client/pd_service_discovery.go

```diff
@@ -732,6 +780,21 @@ func (c *pdServiceDiscovery) GetServiceClient() ServiceClient {
 	return leaderClient
 }
 
+// GetAllServiceClients implments ServiceDiscovery
+func (c *pdServiceDiscovery) GetAllServiceClients() []ServiceClient {
```

Comment:
Maybe return leader and follower addresses respectively could help with the above comment.

Reply:
It's both OK to me. My idea is that we only need one `for loop`, so I just return all clients together